ELJSBridge/Source/Bridge.swift

```diff
@@ -72,17 +75,15 @@ public class Bridge: NSObject {
      */
     public func loadFromFile(filePath: String) throws {
         let filemanager = NSFileManager.defaultManager()
+        
         if filemanager.fileExistsAtPath(filePath) {
-            do {
-                let script = try String(contentsOfFile: filePath, encoding: NSUTF8StringEncoding)
-                do {
-                    try self.load(script)
-                } catch let catchError as NSError {
-                    throw catchError
-                }
-            } catch  {
-                throw NSError(ELJSBridgeError.FileDoesNotExist)
+            guard let script = try? String(contentsOfFile: filePath, encoding: NSUTF8StringEncoding) else {
```

Comment:
Any reason why we would want to mask the error output from `NSString(contentsOfFile:encoding:)` with a generic FailedToLoad error instead of letting the original error propogate? I think the information from the NSString error could be helpful for knowing why the data failed to be read into a string.